Code/GraphMol/FileParsers/CDXMLParser.cpp

```diff
@@ -422,6 +422,16 @@ bool parse_fragment(RWMol &mol, ptree &frag,
       } else if (bond.display == "WedgedHashBegin" ||
                  bond.display == "WedgedHashEnd") {
         bnd->setBondDir(Bond::BondDir::BEGINWEDGE);
+      } else if (bond.display == "Wavy") {
+	switch(bond.getBondType()) {
```

Comment:
maybe should be an if statement?  Should a wavy bond always be a double bond?  There are some cases where it isn't in the test set (these are obviously incorrect)

Reply:
I wouldn't think that it should always be double.
I'm not sure how chemdraw works with this stuff, but in v3k mol files bond CFG=2 is rendered as a crossed bond when applied to a double bond or as a wavy bond if it's single. You definitely need to be able to represent wavy single bonds, but maybe chemdraw does that differently?

Reply:
I think this has the right logic now for wavy bonds.